src/test/java/essenmakanan/recipe/DuplicateRecipeTest.java

```diff
@@ -0,0 +1,78 @@
+package essenmakanan.recipe;
+
+import essenmakanan.command.DuplicateRecipeCommand;
+import essenmakanan.ingredient.Ingredient;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DuplicateRecipeTest {
+    private RecipeList recipes;
+    private Recipe recipe;
+    private Recipe duplicatedRecipe;
+
+    @BeforeEach
+    public void setup() {
+        recipes = new RecipeList();
+        String recipeTitle = "sandwich";
+        String[] ingredients = {"i/bread,2,pc", "i/meat,1,pc"};
+        String[] steps = {"step1", "step2"};
+        recipe = new Recipe(recipeTitle, steps, ingredients);
+        recipes.addRecipe(recipe);
+    }
+
+    @Test
+    public void duplicateRecipe_numericalIndex_expectSimilarRecipe() {
+        DuplicateRecipeCommand command = new DuplicateRecipeCommand(recipes, "1");
+        command.executeCommand();
```

Comment:
should this be in the DuplicateRecipeCommand test?

Reply:
Sure, i will move it